packages/side-nav/src/SideNavGroupCollapsed/SideNavGroupCollapsed.tsx

```diff
@@ -53,14 +53,14 @@ export function SideNavGroupCollapsed({
 
   // compute the entered ul wrapper styles based on the ul height
   useEffect(() => {
-    const ulHeight = ulRef?.current?.getBoundingClientRect().height ?? 0;
+    const ulHeight = ulRef.current?.getBoundingClientRect().height ?? 0;
     transitionStyles['entered'] = css`
       opacity: 1;
       max-height: ${ulHeight + 1}px; // +1 for border
       border-bottom: 1px solid
         ${darkMode ? palette.gray.dark1 : palette.gray.light2};
     `;
-  }, [open, ulRef, darkMode]);
+  }, [open, ulRef.current, darkMode]);
```

Comment:
think this approach may warrant a refactor but not sure it makes sense at this time if this will be deprecated in favor of the new nav. feel free to let me know otherwise though @brookeyalof

Reply:
I'm skeptical of this because refs don't trigger a callback. Could we instead use a callback ref(`useState`)?

Reply:
updated!